contrib/bf-grafana/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.4'
+
+services:
+  blueflood:
+    image: rackerlabs/blueflood
+    environment:
+      CASSANDRA_HOST: cassandra
+      ELASTICSEARCH_HOST: elasticsearch
+    ports:
+          - "19000:19000"
+          - "20000:20000"
+          - "9180:9180"
+
+  cassandra:
+    image: cassandra:2.1
+    ports:
+          - "9042:9042"
+          - "9160:9160"
+          - "7199:7199"
+
+  cassandran:
```

Comment:
having this separate "cassandran" service seems a bit wierd.  i'm guessing you copied that from somewhere.

Reply:
Yep, this file started from https://github.com/rackerlabs/blueflood/blob/master/contrib/blueflood-docker-compose/docker-compose.yml

I'm assuming they wanted to create a little two-node cassandra cluster with the "n" instance. That one should be scalable with a `docker-compose scale cassandran=5` type command...but I haven't tried that yet here.